docs/tutorials/examples.rst

```diff
@@ -290,135 +290,50 @@ zipcode for each user:
-columns and data types.
-
-Example: Create a new stream ``pageviews_reduced``, similar to the
-previous example, but with only a few of all the available fields in the
-Avro data (here, only the two columns ``viewtime`` and ``pageid`` are
-picked).
-
-.. code:: sql
-
-    CREATE STREAM pageviews_reduced (viewtime BIGINT, pageid VARCHAR)
-      WITH (KAFKA_TOPIC='pageviews-avro-topic',
-            VALUE_FORMAT='AVRO');
-
-KSQL allows you to work with streams and tables regardless of their
-underlying data format. This means that you can easily mix and match
-streams and tables with different data formats (e.g. join a stream
-backed by Avro data with a table backed by JSON data) and also convert
-easily between data formats.
-
-Example: Convert a JSON stream into an Avro stream.
-
-.. code:: sql
+-   This example uses a Bash `here string <http://tldp.org/LDP/abs/html/x17837.html>`__ (``<<<``) to run KSQL CLI commands on
+    an explicitly defined KSQL server endpoint.
 
-    CREATE STREAM pageviews_json (viewtime BIGINT, userid VARCHAR, pageid VARCHAR)
-      WITH (KAFKA_TOPIC='pageviews-json-topic', VALUE_FORMAT='JSON');
+    .. code:: bash
 
-    CREATE STREAM pageviews_avro
-      WITH (VALUE_FORMAT = 'AVRO') AS
-      SELECT * FROM pageviews_json;
+        $ ksql http://localhost:8088 <<< "SHOW TOPICS;
+        SHOW STREAMS;
+        exit"
 
-Note how you only need to set ``VALUE_FORMAT`` to Avro to achieve the
-data conversion. Also, KSQL will automatically generate an appropriate
-Avro schema for the new ``pageviews_avro`` stream, and it will also
-register the schema with Confluent Schema Registry.
+-   This example creates a stream from a predefined script (``application.sql``) using the ``RUN SCRIPT`` command and
+    then runs a query by using the Bash `here document <http://tldp.org/LDP/abs/html/here-docs.html>`__ (``<<``) feature.
 
-Running KSQL
-------------
+    .. code:: bash
 
-KSQL supports various :ref:`modes of operation <modes-of-operation>`, including a standalone mode and a client-server mode.
+        $ cat /path/to/local/application.sql
+        CREATE STREAM pageviews_copy AS SELECT * FROM pageviews;
 
-Additionally, you can also instruct KSQL to execute a single statement
-from the command line. The following example command runs the given
-``SELECT`` statement and show the results in the terminal. In this
-particular case, the query will run until 5 records have been found, and
-then terminate.
+    .. code:: bash
 
-.. code:: shell
+        $ ksql http://localhost:8088 <<EOF
+        > RUN SCRIPT '/path/to/local/application.sql';
+        > exit
+        > EOF
 
-    $ ksql http://localhost:8088 --exec "SELECT * FROM pageviews LIMIT 5;"
\ No newline at end of file
+    .. note:: The ``RUN SCRIPT`` command only supports a subset of KSQL CLI commands, such as running persistent queries (CREATE
+              STREAM, CREATE TABLE), and setting configuration options (SET statement). It cannot show output for statements,
```

Comment:
Also add CSAS and CTAS to this.

Reply:
such as running persistent queries (CREATE STREAM AS SELECT, CREATE TABLE AS SELECT), DDL statements (CREATE STREAM, CREATE TABLE) and setting ...

Reply:
And: it is not only hiding the output. It isn’t even executing those other statements.